core/domain/exp_services.py

```diff
@@ -1530,16 +1530,7 @@ def get_user_exploration_data(
             exploration_id, exploration.version, exploration.states))
     for index, state_name in enumerate(exploration.states):
         if classifier_training_jobs[index] is not None:
-            classifier_data = classifier_training_jobs[
-                index].classifier_data
-            algorithm_id = classifier_training_jobs[index].algorithm_id
-            data_schema_version = (
-                classifier_training_jobs[index].data_schema_version)
-            state_classifier_mapping[state_name] = {
-                'algorithm_id': algorithm_id,
-                'classifier_data': classifier_data,
-                'data_schema_version': data_schema_version
-            }
+            state_classifier_mapping[state_name] = classifier_training_jobs[index].to_player_dict()
```

Comment:
Pleaes wrap to 80 chars. You can end the line after the equals sign (with `= (`) and then have a line break and then have an additional closing paren at the end.

Reply:
Done. Thanks for the guide!